tensorflow/core/tpu/tpu_initializer_helper.h

```diff
@@ -26,7 +27,10 @@ namespace tpu {
 // up calls to this function will return true if the lock has been acquired and
 // false if we failed to acquire the lock.
 bool TryAcquireTpuLock();
-
+// This will initialize the TPU library.
+Status InitializeTpuLibrary(void* library_handle);
```

Comment:
Does this one need to be in the header file actually? I think we'll only call FindAndLoadTpuLibrary from other files, is this right?

Reply:
we call TryAcquireTpuLock from profiler as well, but with this InitializeTpuLibrary seems to be defined and used only here. I removed it.

Reply:
Yeah I only meant InitializeTpuLibrary, github just shows a few lines from above as well :)